rules/windows/process_creation/win_susp_explorer.yml

```diff
@@ -0,0 +1,32 @@
+title: Proxy Execution Via Explorer.exe
+id: 9eb271b9-24ae-4cd4-9465-19cfc1047f3e
+description: Attackers can use explorer.exe for evading defense mechanisms
+author: 'Furkan CALISKAN, @caliskanfurkan_, OSCD Community'
```

Comment:
could you please change "OSCD Community" to "oscd.ommunity" or "@oscd_initiative"? 
the thing is that oscd is an initiative, not a community.

the only place where you can see the "community" word near to the "oscd" word is the domain name.
but the only reason I've chosen this domain is that it was cheap and kinda matching the idea — it is a community activity of course, but the global infosec community, not the "oscd community". 

sorry for inconvenience.

Reply:
Done